### PR TITLE
PMM-8460: service software versions

### PR DESCRIPTION
**Pull request by oter**
Ticket: https://jira.percona.com/browse/PMM-8460

API PR: https://github.com/percona/pmm/pull/766
managed PR: https://github.com/percona/pmm-managed/pull/807

FB: https://github.com/Percona-Lab/pmm-submodules/pull/1941

### CODE REVIEW

File: client/version.go [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+	mysqlBin            = "mysql"
+	xtrabackupBin       = "xtrabackup"
+)
+
+var (
+	mysqlVersionRegexp      = regexp.MustCompile("^.*Ver ([!-~]*).*")
+	xtrabackupVersionRegexp = regexp.MustCompile("^xtrabackup version ([!-~]*).*")
+)
+
+func (c *Client) localMySQLVersion() (string, error) {
```
Comment (BupycHuk) [inline client/version.go:43]: Can we extract these 3 methods to separate structure?

Comment (oter) [reply client/version.go:43]: done locally, will push changes after API update.

Comment (oter) [reply client/version.go]: done

**Review by Dasio — APPROVED** [review]

**Review by BupycHuk — APPROVED** [review]

**Review by artemgavrilov — APPROVED** [review]

File: client/version.go [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+import (
+	"github.com/percona/pmm/api/agentpb"
+	"github.com/pkg/errors"
+	"google.golang.org/grpc/status"
+
+	"github.com/percona/pmm-agent/versioner"
+)
+
+func (c *Client) handleVersionsRequest(r *agentpb.GetVersionsRequest) ([]*agentpb.GetVersionsResponse_Version, *status.Status) {
```
[the remark this answers is not in the document]
Comment (oter) [reply client/version.go:27]: fixed

**Pull request merged**